src/planet-4-151612/wordpress/wp-config.php.tmpl

```diff
@@ -8,6 +8,7 @@ define('DB_HOST',     '{{ .Env.WP_DB_HOST}}:{{ .Env.WP_DB_PORT }}');
 
 define('DB_CHARSET',  '{{ .Env.WP_DB_CHARSET }}');
 define('DB_COLLATE',  '{{ .Env.WP_DB_COLLATION }}');
+define('SMARTSHEET_API_KEY', '{{ .Env.SMARTSHEET_API_KEY }}');
```

Comment:
Maybe move it a bit further down and with a short comment, just not to be confused with MySQL settings.